backend/Fusion/Fusion.RestApi/Program.cs

```diff
@@ -54,6 +55,7 @@ private static void ConfigureMiddlewares(WebApplication appBuilder)
         if (appBuilder.Environment.IsDevelopment())
         {
             appBuilder.UseDeveloperExceptionPage();
+            appBuilder.ApplyMigrations();
         }
```

Comment:
Why do we ApplyMigrations in development only?

Reply:
it is not recomended to make auto apply migration in production, because it may cause perfomance issues on startup an d data corruption